docs/API.md

```diff
@@ -650,6 +651,32 @@ some.m('a', 'b')
 console.log('Calls to m: ', mMock.mock.calls)
 ```
 
+### `mockFn.mockImplementationOnce(fn)`
+Accepts a function that will be used as an implementation of the mock for one call to the mocked function. Can be chained so that multiple function calls produce different results.
+
+```
+var myMockFn = jest.fn()
+  .mockImplementationOnce(cb => cb(null, true))
+  .mockImplementationOnce(cb => cb(null, false));
+
+myMockFn((err, val) => console.log(val));
+> true
+
+myMockFn((err, val) => console.log(val));
+> false
+```
+
+When the mocked function runs out of implementations defined with mockImplementationOnce, it will execute the default implementation set with jest.fn (if it is defined):
```

Comment:
Can you change this to "`jest.fn(() => defaultValue)` or `.mockImplementation(() => defaultValue)` if they were called."

I think that might be clearer. What do you think?